homeassistant/components/systemmonitor/sensor.py

```diff
@@ -56,66 +60,158 @@
 SENSOR_TYPE_UOM = 1
 SENSOR_TYPE_ICON = 2
 SENSOR_TYPE_DEVICE_CLASS = 3
-SENSOR_TYPE_MANDATORY_ARG = 4
+SENSOR_TYPE_STATE_CLASS = 4
+SENSOR_TYPE_MANDATORY_ARG = 5
 
 SIGNAL_SYSTEMMONITOR_UPDATE = "systemmonitor_update"
 
-# Schema: [name, unit of measurement, icon, device class, flag if mandatory arg]
-SENSOR_TYPES: dict[str, tuple[str, str | None, str | None, str | None, bool]] = {
-    "disk_free": ("Disk free", DATA_GIBIBYTES, "mdi:harddisk", None, False),
-    "disk_use": ("Disk use", DATA_GIBIBYTES, "mdi:harddisk", None, False),
+# Schema: [name, unit of measurement, icon, device class, state class, flag if mandatory arg]
+SENSOR_TYPES: dict[
+    str, tuple[str, str | None, str | None, str | None, str | None, bool]
+] = {
+    "disk_free": (
```

Comment:
Instead of adding things to this tuple, please add https://github.com/home-assistant/core/pull/53653/files